cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -2,7 +2,10 @@
 Distributed training using Horovod on Spark
 -------------------------------------------
 """
-
+# %%
+# Importing the Libraries
+# ========================
```

Comment:
I don't think we need this, what do you think, @SmritiSatyanV?

Reply:
Sure, I will remove it..